code/__DEFINES/mobs.dm

```diff
@@ -480,6 +480,7 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define XENO_GARGOYLE_DETECTION_COOLDOWN 30 SECONDS
 #define XENO_GARGOYLE_DETECTION_RANGE 10//How far gargoyles can detect hostiles
 #define XENO_RESTING_COOLDOWN 2 SECONDS
+#define XENO_UNRESTING_COOLDOWN 1 SECONDS
```

Comment:
```suggestion
#define XENO_UNRESTING_COOLDOWN 0.5 SECONDS
```

Reply:
shortening this because 1 second is actually quite long, considering this is just to cover misclicks